src/transformers/models/persimmon/modeling_persimmon.py

```diff
@@ -452,7 +452,6 @@ def forward(
     "The bare Persimmon Model outputting raw hidden-states without any specific head on top.",
     PERSIMMON_START_DOCSTRING,
 )
-# Copied from transformers.models.llama.modeling_llama.LlamaPreTrainedModel with Llama->Persimmon
```

Comment:
Hmmm a bit annoying that we lose the pairing between the two, given the two are nearly equivalent should we just enable flash attention for this model as well?

Reply:
Hmm I tried to add it and it led to tests failing for batched generation I need to dig more, maybe we can leave it as it is for a v1, what do you think?

Reply:
yes sgtm